vividus-engine/src/main/java/org/vividus/steps/ComparisonRule.java

```diff
@@ -40,6 +40,14 @@ public <T extends Comparable<T>> Matcher<T> getComparisonRule(T variable)
             return Matchers.lessThanOrEqualTo(variable);
         }
     },
+    AT_MOST("at most")
```

Comment:
```suggestion
    AT_MOST("<=")
```
?

Reply:
this was done to return LESS_THAN_OR_EQUAL_TO value in `fromString` method  for '<=' sign regardless of the order of declaration of elements in enum

Reply:
ok, then just add `null` alias, and add processing of `null`-s